package com.company;

public class Example6 {
    Example6(){
        /* Пример с ошибкой иерархии исключений
        try {
            System.out.println("0");
            throw new NullPointerException("ошибка");
        } catch (ArithmeticException e) {
            System.out.println("1" );
        }catch (Exception e) {
            System.out.println("2" );
        } catch (RuntimeException e) {
            System.out.println("3" );
        }
        System.out.println("4");
         */
    }
}
